src/particleContainer/adapter/RDFCellProcessor.cpp

```diff
@@ -74,6 +88,13 @@ void RDFCellProcessor::processCellPair(ParticleCell& cell1, ParticleCell& cell2,
 					double dd = molecule2.dist2(molecule1, dummy);
 					if (dd < _cutoffRadiusSquare) {
 	                    _rdf->observeRDF(molecule1, molecule2, dd);
+						if (_rdf->doARDF() == true) {
```

Comment:
simplify this statement
```suggestion
						if (_rdf->doARDF()) {
```